course_discovery/apps/api/tests/test_serializers.py

```diff
@@ -1049,6 +1075,10 @@ def test_data(self):
 
 
 class ContainedCourseRunsSerializerTests(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.edx_org_short_name = 'edX'
```

Comment:
Why we need to add `setUp` method to specifically add `edx_org_short_name` class method when we are only going to use it in one test method `test_filter_contained_course_runs_on_edx_org_short_name` ?

Set a variable in the method `test_filter_contained_course_runs_on_edx_org_short_name` for this.

Reply:
Done